yt_dlp/extractor/radlive.py

```diff
@@ -0,0 +1,381 @@
+import json
+import re
+import textwrap
+import urllib.request
+from datetime import datetime
+
+from ..utils import ExtractorError
+from .common import InfoExtractor
+
+
+class RadLiveIE(InfoExtractor):
+    IE_NAME = 'radlive'
+    _VALID_URL = r'https?://(?:www\.)?rad\.live/content/(?P<content_type>feature|episode)/(?P<id>[a-f0-9-]+)'
```

Comment:
isn't it better to make is `a-z` to be future proof?

Reply:
Maybe, though it’s supposed to be a hexadecimal UUID.

Reply:
it can be left as it is then ig